lib/fizzy/capi.cpp

```diff
@@ -21,6 +21,38 @@ inline const fizzy::Module* unwrap(const FizzyModule* module) noexcept
     return reinterpret_cast<const fizzy::Module*>(module);
 }
 
+static_assert(sizeof(FizzyValueType) == sizeof(fizzy::ValType));
```

Comment:
Hm, perhaps there should be such an assert for `FizzyValue` too.

Reply:
There `bit_cast` already does this.